public-ui/src/app/views/ehr-view/ehr-view.component.css

```diff
@@ -7,3 +7,6 @@
 .white-card-box.top-stat {
     padding: 1rem;
 }
+.flex-items-xs-right {
```

Comment:
Please don't put custom styles on built classes used by grid system. make your own class and add it to it.  

These built in classes are meant to do a very particular thing and are not intended to customize unless you want that customization to happen everywhere on the page. Which in general we dont.